packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -49,7 +49,13 @@ export class ClusterReachability extends EventsScope {
   private srflxIceCandidates: RTCIceCandidate[] = [];
   public readonly isVideoMesh: boolean;
   public readonly name;
-  public readonly reachedSubnets: Set<string> = new Set();
+  public readonly reachedSubnets: Set<{
+    protocol: string;
+    serverIp: string;
+    reachable: string;
```

Comment:
why do we need `reachable` here? the name of this set is `reachedSubnets` so they're all always reachable.